src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -29,6 +29,12 @@ public class ActorRuntime
 
         private ActorSettings actorSettings;
 
+        internal static JsonSerializerOptions JsonSerializerOptions { get; } = new JsonSerializerOptions()
```

Comment:
This should be private and non-static - or public and non-static if it needs to exposed for other consumers. Generally if the user configures it, then the user should be able to access it (public).